eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
         - (Alpine.311.Amd64.Open)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.11-helix-08e8e40-20200107182408
       - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
-        - (Alpine.38.Amd64.Open)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.8-helix-09ca40b-20190620184125
+        # https://github.com/dotnet/runtime/issues/1871
+        # - (Alpine.38.Amd64.Open)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.8-helix-09ca40b-20190620184125
```

Comment:
@jaredpar Didn't you intend to disable arm64 runs?

Reply:
Silly mistake on my part. Thanks for catching.